.github/workflows/brownie.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 16.x ]
+        node-version: [ 17.x ]
```

Comment:
I recommend using a stable version. Odd node-versions aren't considered stable versions.
I think ganache-cli issue is related to node beta version.